samples/news/News.UITests/News.UITests.csproj

```diff
@@ -49,6 +49,10 @@
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\News\News.csproj">
```

Comment:
this reference is a new one to access common constants (list of known categories and sources) located in the shared core project